src/test/resources/mapper/core/role_limit/map_limits/core_limits.json

```diff
@@ -0,0 +1,50 @@
+{
+  "deployment1": {
+    "minute": 9223372036854775807,
+    "day": 10,
+    "week": 9223372036854775807,
+    "month": 9223372036854775807,
+    "requestHour": 9223372036854775807,
+    "requestDay": 9223372036854775807
+  },
+  "deployment2": {
+    "minute": 9223372036854775807,
+    "day": 5,
+    "week": 9223372036854775807,
+    "month": 9223372036854775807,
+    "requestHour": 9223372036854775807,
+    "requestDay": 9223372036854775807
+  },
+  "deployment3": {
+    "minute": 9223372036854775807,
+    "day": 9223372036854775807,
+    "week": 9223372036854775807,
+    "month": 9223372036854775807,
+    "requestHour": 9223372036854775807,
+    "requestDay": 9223372036854775807
+  },
+  "deployment4": {
+    "minute": 9223372036854775807,
```

Comment:
i'd left one limit empty (e.g. "deployment4": {}) to test default value mapping. and it's main limits declaration in real worlds core configs.